.travis.yml

```diff
@@ -18,20 +23,28 @@ addons:
       - libscalapack-openmpi-dev
       - libarpack2-dev
       - pkg-config
+  homebrew:
+    packages:
+      - arpack
+      - python@3.8
```

Comment:
Maybe explicitly install GCC with ``gcc@9`` or ``gcc@10`` as well?

Reply:
Valid point, indeed. I just tried it. :-1:  Now I really tend to leave away all homebrew package installation and reduce testing instead. (Adding `gcc@10` upgrades to from gcc 10.2 to gcc 10.2_1. Adding `arpack` installs among others latest python 3.9 (why?!) colliding with python 3.8 already present, causing homebrew exiting with an error...) I don't see an easy way to establish a reliable testing framework if homebrew is invoked...